releasenotes/notes/fix-botocore-bug-when-empty-poll-disabled-config-set-033d13dee074ec23.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    botocore: fix bug that wrapped function results/errors are not returned for SQS and Kinesis when `DD_BOTOCORE_EMPTY_POLL_ENABLED=false` or 
```

I think the current release note is hard to more ddtrace users to understand. 

We can simplify a bit.

Ex:
```suggestion
    botocore: Fixes bug where SQS and Kinesis results and errors were not recorded when `DD_BOTOCORE_EMPTY_POLL_ENABLED=false`.
```